spotinst/resource_spotinst_stateful_node_azure_v3_test.go

```diff
@@ -0,0 +1,1226 @@
+package spotinst
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	azurev3 "github.com/spotinst/spotinst-sdk-go/service/stateful/providers/azure"
```

Comment:
no need "azurev3" , import the azure library from go-sdk and use it as is without the "azurev3" alias